controllers/operatorconfigmap_controller.go

```diff
@@ -735,6 +740,36 @@ func (c *OperatorConfigMapReconciler) reconcileSubscription() error {
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileCSIAddonsSubscription() error {
+	subs := &opv1a1.SubscriptionList{}
+	if err := c.list(subs, client.InNamespace(c.OperatorNamespace)); err != nil {
+		return fmt.Errorf("failed to list subscriptions")
+	}
+
+	clientSub := utils.Find(subs.Items, func(sub *opv1a1.Subscription) bool {
+		return sub.Spec.Package == "ocs-client-operator"
+	})
+	if clientSub == nil {
+		return fmt.Errorf("failed to find subscription with ocs-client-operator package")
+	}
+
+	addonsSub := utils.Find(subs.Items, func(sub *opv1a1.Subscription) bool {
+		// package naming of csi-addons differs in u/s & d/s, however 'csi-addons' exists as substring in the name
+		return strings.Contains(sub.Spec.Package, "csi-addons")
```

Comment:
Everything else LGTM, can we use same name for both upstream and downstream and instead of Contains check have a complete string matching check here.

Reply:
Why we can't just check both one after the other with a logical or between them?

Reply:
ack, I'll change it to exact match for `csi-addons` or `odf-csi-addons-operator`

Reply:
done.